PHPCI/Model/Base/ProjectBase.php

```diff
@@ -148,6 +151,9 @@ class ProjectBase extends Model
             'type' => 'int',
             'length' => 11,
         ),
+        'archive' => array(
+            'type' => 'boolean',
```

Comment:
@zviryatko The best practice is:

1. Make a migration.
2. Run it.
3. Run `./console db:generate`

That'll update the database, and then update your models to reflect the changes to the database. 

The *Base.php files should never be edited by hand.